src/SearchParameters/index.js

```diff
@@ -21,11 +21,18 @@ var pluck = require('lodash/collection/pluck');
 var defaults = require('lodash/object/defaults');
 var merge = require('lodash/object/merge');
 var deepFreeze = require('../functions/deepFreeze');
+var valToNumber = require('../functions/valToNumber');
 
 var filterState = require('./filterState');
 
 var RefinementList = require('./RefinementList');
 
+function findArray(vs, v1) {
```

Comment:
A bit hard to quickly grasp the function signature, we have three variables named vs, v1 and v. Can you make them explicit?